airflow/utils/cli_action_loggers.py

```diff
@@ -94,8 +94,11 @@ def default_action_log(log, **_):
     :param **_: other keyword arguments that is not being used by this function
     :return: None
     """
-    with create_session() as session:
-        session.add(log)
+    try:
+        with create_session() as session:
+            session.add(log)
+    except Exception as error:
```

Comment:
Is it possible to specify what exception exactly we want to catch here?

Reply:
It is, though I don't think it's worth it. An SQLAchemy exception is basically the only thing that could pop out of that `session.add`, and filtering by error message doesn't seem relevant to me.